ui/pages/campaign/[address].tsx

```diff
@@ -424,42 +518,26 @@ const CampaignContent: FC<CampaignContentProps> = ({
               <p className="text-light text-sm">
                 pledged out of {goal.toLocaleString()} {payTokenSymbol} goal.
               </p>
-              {/* TODO: Display supporters. */}
+              {/* TODO: Display backers. */}
               {/* <h3 className="mt-6 text-green text-3xl">
-                {supporters.toLocaleString()}
+                {backers.toLocaleString()}
                 </h3>
-                <p className="text-light text-sm">Supporters</p> */}
-
-              {/* <h3 className="mt-6 text-green text-3xl">
-                {supply.toLocaleString()}
-                </h3>
-                <p className="text-light text-sm">Total Supply</p> */}
+                <p className="text-light text-sm">Backers</p> */}
+
+              {!!campaignGovTokenPercentage && !!govTokenSymbol && (
+                <>
+                  <h3 className="mt-6 text-green text-3xl">
+                    {prettyPrintDecimal(campaignGovTokenPercentage, 2)}%{" "}
+                    governance
+                  </h3>
```

Comment:
Once the campaign completes it will start paying out governance tokens to backers and this percentage will no longer be correct. Longer term we should keep state somewhere about the initial pay-in but to start how about hiding this for funded campaigns?